packages/dataviews/src/types.ts

```diff
@@ -156,6 +167,19 @@ export type Form = {
 	visibleFields?: string[];
 };
 
+export type DataFormProps< Item > = {
```

Comment:
Putting this one here, means we should also put the DataViewsProps here rather than collocating by the file. So far for this kind of components we kept the types collocated though.

Reply:
In https://github.com/WordPress/gutenberg/pull/64171/commits/40043fc1cb3b02d925154651f882fd65cc6eab60 I collocated `DataFormProps` back into the component.

`DataFormControlProps` still lives in the types file: it's required there and there are multiple places where it's used (every type definition). Is there a better place for this one?